sdk/core/Azure.Core/samples/Serialization.md

```diff
@@ -1,34 +1,48 @@
-# Azure.Core public serialization samples
+# Azure.Core Public Serialization Guide
 
-## Using explicit cast
+The latest updates to the `Azure.Core` library have simplified serializing and deserializing of all public Azure models. With the addiion of the `ModelSerializer` class, we provide a public interface that exposes the previously internal serialization code so customers can access it without needing to use reflection or write their own translation. The recommended serialization approach is to use the static `ModelSerializer` class. The `ModelSerializer` class also has `ModelSerializerOptions` which allows the user to set the serialization `ModelSerializerFormat` type (`Json` or `Wire`) and provide custom Serializer types for specific models.
 
-When using protocol methods for advanced handling of RequestContext it is still possible to use the strongly typed models.
-There is an explicit cast operator that can be used to convert the protocol Response to the strongly typed model.
-There is also an explicit cast operator that can be used to convert the strongly typed model to the protocol RequestContent.
+## Key Concepts
+
+- [ModelSerializerFormat description]("modelserializerformat-description")
```

Comment:
This does not exist.  

When adding it, I'd suggest that you place it after the basic "how to use this" sections.   Our goal is to let folks who want to scan and get started quickly find the "how to" content and then come back for the "how do i?" and "how does this work" content.